demo/index.html

```diff
@@ -16,45 +16,70 @@
   <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
   <link rel="import" href="../iron-ajax.html">
   <link rel="import" href="../../iron-image/iron-image.html">
-  <link rel="import" href="../../paper-styles/demo-pages.html">
-  <style>
-    iron-image {
-      background-color: lightgray;
-      margin: 1em;
-    }
-    .horizontal-section {
-      max-width: 300px;
-      margin-bottom: 24px;
-    }
-  </style>
 </head>
 <body unresolved>
   <h1>Video Feed</h1>
-  <div class="horizontal-section-container">
-    <template is="dom-bind" id="app">
+  <dom-module id="iron-ajax-demo">
+    <template>
+      <custom-style include="iron-flex">
+        <style>
+          iron-image {
+            background-color: lightgray;
+            margin: 1em;
+          }
+          .horizontal-section-container {
+            display: -ms-flexbox;
+            display: -webkit-flex;
+            display: flex;
+            -ms-flex-pack: center;
+            -webkit-justify-content: center;
+            justify-content: center;
+            -ms-flex-wrap: wrap;
+            -webkit-flex-wrap: wrap;
+            flex-wrap: wrap;
+          }
+          .horizontal-section {
+            background-color: white;
+            padding: 24px;
+            margin-right: 24px;
+            min-width: 200px;
+            box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14),
+                        0 1px 5px 0 rgba(0, 0, 0, 0.12),
+                        0 3px 1px -2px rgba(0, 0, 0, 0.2);
+            max-width: 300px;
+            margin-bottom: 24px;
+          }
+        </style>
+      </custom-style>
+      
       <iron-ajax auto
           url="https://www.googleapis.com/youtube/v3/search"
           params='{"part":"snippet", "q":"polymer", "key": "AIzaSyAuecFZ9xJXbGDkQYWBmYrtzOGJD-iDIgI", "type": "video"}'
           handle-as="json"
           last-response="{{ajaxResponse}}"></iron-ajax>
 
+      <div class="horizontal-section-container">
         <template is="dom-repeat" items="[[ajaxResponse.items]]">
           <div class="horizontal-section">
             <h2><a href="[[url(item.id.videoId)]]" target="_blank">[[item.snippet.title]]</a></h2>
             <iron-image src="[[item.snippet.thumbnails.high.url]]" width="256" height="256" sizing="cover" preload fade></iron-image>
             <p>[[item.snippet.description]]</p>
           </div>
         </template>
-
+      </div>
     </template>
-  </div>
 
-  <script>
-    var app = document.querySelector('#app');
+    <script>
```

Comment:
If you want this to run in FF & friends, you need to wrap it in a `window.addEventListener('WebComponentsReady')`